1-exercises/D-array-filter/exercise.js

```diff
@@ -6,17 +6,23 @@
   - Do not edit any of the existing code
 */
 
-let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
+let pairsByIndexRaw = [
+    [0, 3],
+    [1, 2],
+    [2, 1], null, [1], false, "whoops"
+];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw.filter(
+    (element) => !!element && element.length === 2
```

Comment:
Will `!!element && element.length` work if the value is `true`?
ie if it passes the first 'gate' (`!!element`) will the second check error? (ie `true.length`).
Maybe instead of `!!element` it's better to check if the element is an array?